docs/nuget-org/nuget-org-faq.yml

```diff
@@ -60,33 +60,55 @@ sections:
           It is possible to share dependencies between multiple packages, making the total download size for consumers of your NuGet packages smaller.
           
           Dependencies are mostly static and never change. When fixing a bug in code, the dependencies may not need to be updated. If you bundle dependencies, you end up reshipping larger packages every time. By splitting NuGet packages into related dependencies, upgrades are much more fine-grained for consumers of your package.
-          
+
   - name: NuGet.org not accessible
     questions:
-      - question: Why can't I download packages from or upload packages to NuGet.org?
+      - question: What TLS version and cipher suites does NuGet.org support?
         answer: |
-          First, make sure you're using the latest versions of NuGet. If that version continues to fail, [contact support](https://www.nuget.org/policies/Contact) and provide additional connection troubleshooting information including:
-          
-          - The version of NuGet you're using
+          NuGet.org supports TLS 1.2 and the following cipher suites:
+          - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+          - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+          - TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+          - TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+
+      - question: Why can't I restore packages from NuGet.org?
+        answer: |
+          If you meet transient failures when restoring packages from NuGet.org, we suggest you:
+          - use [V3 API endpoint](https://learn.microsoft.com/nuget/nuget-org/overview-nuget-org#api-endpoint-for-nugetorg) to restore packages from NuGet.org.
```

Comment:
Do you think, at the very end, we can suggest using V2 as a workaround? I'm not happy about the idea, but we've had situations in the past where V2 worked for folks when V3 did not.

Reply:
I will update the support handbook soon. How about we put the suggestion of trying V2 feed there?

The reason why I mention V3 feed here is that we have a much better control on V3 traffic. If we say that the outage scope is very limited (like ISP related) and we would not like to run the failover, we can suggest that customers use V2 feed by emails.

Reply:
I have removed the suggestion of using V3 feed in this document and will update the support handbook to mention about using V2 feed when V3 feed is under the degradation.